bin/build-sass.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-node-sass \
+npx --yes node-sass \
   --output-style=compressed \
   source/_sass/all.scss \
-  output_dev/css/all.css
+  output_dev/css/all.css || true
```

Comment:
Why would you make it like this? Having a failure should be reported.
Also, is NPX available inside the container?

Reply:
NPX is available within this container as it's the standard docker nodejs. I pipe to true which does not change logging, but is unfortunate because despite succeeding node-sass was emitting warnings, and therefore giving non-zero error code.